src/components/details/Detailspage.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { getCompanyDetails } from '../../redux/companies-list';
+import './Detailspage.css';
+
+const Detailspage = () => {
+  const { companyDetails } = useSelector((state) => state.companies);
+  const dispatch = useDispatch();
+  const { smbl } = useParams();
+  useEffect(() => {
+    dispatch(getCompanyDetails(smbl));
+  }, [dispatch]);
+  // console.log(`companyDetails ${companyDetails}`);
```

Comment:
- [ ] Kindly remove commented code.

Reply:
- [ ] Kindly make sure to remove the commented code.